src/ServiceControl/Operations/CheckRemotes.cs

```diff
@@ -69,14 +69,17 @@ static async Task CheckSuccessStatusCode(HttpClient client, RemoteInstanceSettin
             {
                 var response = await client.GetAsync(remoteSettings.ApiUri, cancellationToken).ConfigureAwait(false);
                 response.EnsureSuccessStatusCode();
+                remoteSettings.TemporarilyUnavailable = false;
             }
             catch (HttpRequestException e)
             {
-                throw new TimeoutException($"Remote at '{remoteSettings.ApiUri}' doesn't seem to be available. Reason: {e.Message}", e);
+                remoteSettings.TemporarilyUnavailable = true;
+                throw new TimeoutException($"Remote at '{remoteSettings.ApiUri}' doesn't seem to be available. Remote instance will be temporarily disabled. Reason: {e.Message}", e);
```

Comment:
```suggestion
                throw new TimeoutException($"The remote instance at '{remoteSettings.ApiUri}' doesn't seem to be available. It will be temporarily disabled. Reason: {e.Message}", e);
```